packages/augur-core/source/contracts/legacy_reputation/OldLegacyRepToken.sol

```diff
@@ -108,27 +108,32 @@ contract OldLegacyReputationToken is DelegationTarget, ITyped, Initializable, Va
     }
 
     function transferFrom(address _from, address _to, uint _value) public whenNotMigratingFromLegacy returns (bool) {
-        return super.transferFrom(_from, _to, _value);
+        require(_value <= allowed[_from][msg.sender], "Not enough funds allowed");
+        allowed[_from][msg.sender] = allowed[_from][msg.sender].sub(_value);
+        balances[_from] = balances[_from].sub(_value);
+        balances[_to] = balances[_to].add(_value);
+        emit Transfer(_from, _to, _value);
+        return true;
     }
 
     function trustedUniverseTransfer(address _source, address _destination, uint256 _attotokens) public whenNotMigratingFromLegacy afterInitialized returns (bool) {
         require(IUniverse(msg.sender) == universe);
-        return internalTransfer(_source, _destination, _attotokens);
+        return internalTransfer(_source, _destination, _attotokens, true);
     }
 
     function trustedMarketTransfer(address _source, address _destination, uint256 _attotokens) public whenNotMigratingFromLegacy afterInitialized returns (bool) {
         require(universe.isContainerForMarket(IMarket(msg.sender)));
-        return internalTransfer(_source, _destination, _attotokens);
+        return internalTransfer(_source, _destination, _attotokens, true);
```

Comment:
Could probably save gas to make this false? But there are cases where the owner might want to be notified their tokens are being sent AND the owner is the one originating the tx so it isn't a risk to others? This implementation is probably the most correct, but does cost extra gas.

No change necessary, just noting it.